experimental/framework/data-objects/tsconfig.json

```diff
@@ -1,11 +1,9 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": "../../../common/build/build-common/tsconfig.node16.json",
 	"exclude": ["dist", "node_modules"],
 	"compilerOptions": {
-		"module": "esnext",
```

Comment:
This was only building esnext before. For both dist and lib. If example included it, there could be wierd imports. Perhaps this clears up some things!